<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditImportanceForm</class>
 <widget class="QDialog" name="EditImportanceForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Importance</string>
  </property>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset resource="PropertyResultImportance.qrc">
     <normaloff>:/Qtres/Cancel.png</normaloff>:/Qtres/Cancel.png</iconset>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>171</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Importance</string>
   </property>
   <widget class="QRadioButton" name="radioButtonVeryHigh">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>17</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Very high (10)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonHigh">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>37</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>High(50)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonNormal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>57</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Normal (100)</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonNotSoHigh">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>77</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Not so high (200)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOK">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
    <property name="icon">
     <iconset resource="PropertyResultImportance.qrc">
      <normaloff>:/Qtres/OK.png</normaloff>:/Qtres/OK.png</iconset>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>0</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Level</string>
   </property>
   <widget class="QSpinBox" name="spinBoxLevel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>12</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Importance level</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Smaller number is higher</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOKLevel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
    <property name="icon">
     <iconset resource="PropertyResultImportance.qrc">
      <normaloff>:/Qtres/OK.png</normaloff>:/Qtres/OK.png</iconset>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="PropertyResultImportance.qrc"/>
 </resources>
 <connections/>
</ui>
